.pre-commit-config.yaml

```diff
@@ -52,7 +52,6 @@ repos:
             (?x)^(
                 .*/?\.*.(tar.gz)
             )$
-    -   id: check-jjbb
```

Comment:
Remove the dependency with the internal docker registry, as the docker image is in the private repo and vault/docker login with secrets won't work for PRs.